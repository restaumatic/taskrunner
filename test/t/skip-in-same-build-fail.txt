# Within one build we don't run a task twice.

go() {
  taskrunner -n foo sh -c '
    echo "Executing"
    exit 1
  '
}

go || echo "failed 1"
go || echo "failed 2" # should not run again, should return failure status
