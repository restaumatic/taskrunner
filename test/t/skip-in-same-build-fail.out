-- output:
[foo] stdout | Executing
[toplevel] stdout | failed 1
[toplevel] stdout | failed 2
